docker/archimedes-base/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN apt-get update \
       gnupg \
       bash-completion \
       git \
+      tk \
```

Comment:
Ran into an issue where poetry install was throwing this as missing for one of the dependencies...not sure if the underlying image changed at some point?